app/src/main/java/com/example/karam/ContractorPostLogin.java

```diff
@@ -0,0 +1,58 @@
+package com.example.karam;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.appcompat.widget.Toolbar;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.Button;
+import android.widget.SearchView;
+import android.widget.Toast;
+
+public class ContractorPostLogin extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.contractor_post_login);
+        assignListenerToViews();
+        Toolbar myToolbar = (Toolbar) findViewById(R.id.my_toolbar);
+        setSupportActionBar(myToolbar);
+    }
+
+    private void assignListenerToViews() {
+        SearchView laborerSearch = (SearchView) findViewById(R.id.search);
+        laborerSearch.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                startActivity(new Intent(ContractorPostLogin.this, ContractorPostLoginSearch.class));
+            }
+        });
+    }
+    @Override
```

Comment:
There must be an option to format the code, use that to automate simple fomatting. 
There should be a line between line 34 and 35